package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "word-aligner",
-  "version": "0.4.0",
+  "version": "0.5.0",
```

Comment:
It qualifies as a major version change, sorry I probably confused you by saying v5. It should actually be v1.